web/views/FcRequestsTrack.vue

```diff
@@ -65,18 +65,42 @@
 
         <v-divider></v-divider>
 
-        <v-card-text class="fc-data-table-requests-wrapper overflow-y-hidden pa-0">
+        <v-card-text
+          class="fc-data-table-requests-wrapper flex-grow-1 flex-shrink-1 pa-0">
           <FcDataTableRequests
             v-model="selectedItems"
             aria-labelledby="heading_track_requests_requests"
             :columns="columns"
             :has-filters="hasFiltersRequest"
+            :height="heightTable"
             :items="items"
+            :items-per-page.sync="itemsPerPage"
             :loading="loading"
-            :sort-by.sync="sortBy"
-            :sort-desc.sync="sortDesc"
+            :page.sync="page"
+            :server-items-length="items.length"
```

Comment:
do we need `server-items-length` here?  we're handling all the pagination outside `<FcDataTableRequests>` anyways...